### PR TITLE
Build a Serverless Voice Conversational Chatbot

### PR DESCRIPTION
**Pull request by rbgo404**
PR for "Build a Serverless Voice Conversational Chatbot"

### CODE REVIEW

**Comment by rbgo404** [comment]
@AGInfer Please Check

**Comment by AGInfer** [comment]
@nickaggarwal please check the code once

File: cookbook/serverless-voice-chatbot.mdx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+```
+Here's the custom runtime for the application:
+```yaml
+build:
+  system_packages:
+    - "libssl-dev"
+  python_packages:
+    - "torchaudio==2.2.1"
+    - "soundfile==0.12.1"
+    - "git+https://github.com/suno-ai/bark.git@refs/pull/391/head"
```
Comment (nickaggarwal) [inline cookbook/serverless-voice-chatbot.mdx:119]: Is there a stable version possible ?

File: cookbook/serverless-voice-chatbot.mdx [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+1. __Ease of Use:__ Forget the complexities of infrastructure management. With Inferless, you simply bring your model, and within minutes, you have a working endpoint. Deployment is hassle-free, without the need for in-depth knowledge of scaling or infrastructure maintenance.
+2. __Cold-start Times:__ Inferless's unique load balancing ensures faster cold-starts. Expect around 28.60 seconds to process each queries, significantly faster than many traditional platforms.
+3. __Cost Efficiency:__ Inferless optimizes resource utilization, translating to lower operational costs. Here's a simplified cost comparison:
+
+### Scenario 1
+You are looking to deploy a Voice Conversational Chatbot application for processing 100 queries.<br />
+
+__Parameters:__
+- __Total number of queries:__ 100 daily.<br />
+- __Inference Time:__ All models are hypothetically deployed on A100 80GB, taking 28.60 seconds of processing time and a cold start overhead of 20.72 seconds.<br />
```
Comment (nickaggarwal) [inline cookbook/serverless-voice-chatbot.mdx:148]: can we deploy this on A10G

File: cookbook/serverless-voice-chatbot.mdx (hunk 1 above)
Comment (rbgo404) [reply cookbook/serverless-voice-chatbot.mdx:119]: No, they don't have a stable version.

File: cookbook/serverless-voice-chatbot.mdx (hunk 2 above)
Comment (rbgo404) [reply cookbook/serverless-voice-chatbot.mdx:148]: @nickaggarwal I have to check that, I have tested it on A100 machine only.

**Pull request merged**